examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,470 @@
+   "metadata": {},
+   "source": [
+    "## Packages and imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93cdcd4d",
+   "metadata": {},
+   "source": [
+    "We'll begin by installing the relevant analytics packages that we'll use for our examples. We'll need:\n",
+    "* `arrow`\n",
+    "* `ggplot2`\n",
+    "* `jsonlite`\n",
+    "* `geojsonio`\n",
+    "* `sf`\n",
+    "* `tidyverse`\n",
+    "\n",
+    "which you can install individually by running `install.packages(arrow)`, etc. These packages allow us to load parquet data, interact with and format data the way we want, and visualize our result. Additionally, we'll need two packages specific to accessing Microsoft's Azure platform: `AzureStor` and `AzureRMR`.\n",
+    "\n",
+    "While common practice is to use the `rgdal` package for spatial analytics purposes (see last example), `rgdal` will reach its end-of-supported lifecycle in early 2023. Hence, we recommend the new industry standard `sf` package (standing for \"simple features\")."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f2e39b1c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import relevant packages\n",
+    "library(arrow)\n",
+    "library(jsonlite)\n",
+    "library(AzureStor)\n",
+    "library(AzureRMR)\n",
+    "library(sf)\n",
+    "library(geojsonio)\n",
+    "library(tidyverse)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "acaf7238",
+   "metadata": {},
+   "source": [
+    "## Access and download data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0871e7a",
+   "metadata": {},
+   "source": [
+    "Now that we've got our package environment set up, let's load the data. This notebook demonstrates the important fact that the output of the PHDI Building Blocks pipeline are cloud-hosted files that are accessible locally for all manner of analytics. Let's access the Storage Account and pull down the parquet data to read into R."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d76d4d85",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Configure container access to Azure storage\n",
+    "az <- create_azure_login(tenant=\"9ce70869-60db-44fd-abe8-d2767077fc8f\", host=\"https://storage.azure.com/\")\n",
+    "token <- az$token$credentials$access_token\n",
+    "adls_ep <- adls_endpoint(\"https://pitestdatasa.dfs.core.windows.net/\", token=token)\n",
+    "container <- storage_container(adls_ep, \"gold\")\n",
+    "\n",
+    "# Initialize some tempfiles to buffer data into\n",
+    "phdi_conditions = tempfile()\n",
+    "phdi_organizations = tempfile()\n",
+    "phdi_compositions = tempfile()\n",
+    "phdi_patients = tempfile()\n",
+    "\n",
+    "# Perform storage download, copying data into the temp files\n",
+    "storage_download(container, \"Condition.parquet\", phdi_conditions)\n",
+    "storage_download(container, \"organization.parquet\", phdi_organizations)\n",
```

Comment:
Why is there a difference in capitalization in the parquet files?

Reply:
These file names are generated by the code we have in `main`. If this is an actual problem,  let's solve the root cause and not band-aid the symptom.

Reply:
I think it was your notebook that generated them but it might be corrected now

Reply:
That's true that these files were created from my notebook. 🤔 A lot of the code I had in that notebook was taken from the code in `main`, though, which is why my first response was that the issue is likely there. I'll look through the notebook and see if the problem exists there. My point was that we should ensure this isn't being unintentionally done in the code somewhere before we try to "fix" it by changing the name in the storage container.